demo/full/scripts/components/GitHubButton.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from "react";
+import { render } from "github-buttons";
+
+const {
+  useEffect,
+  useRef,
+} = React;
+
+const GitHubButton = ({
+  href,
+  ariaLabel,
+  dataColorScheme = "no-preference: dark_high_contrast; light: dark_high_contrast; dark: dark_high_contrast;",
+  dataIcon,
+  dataShowCount,
+  dataSize = "large",
+  dataText,
+  title,
+  children,
+}: {
+  href: string;
+  ariaLabel?: string;
+  dataColorScheme?: string;
+  dataIcon?: string;
+  dataShowCount?: boolean | string;
+  dataSize?: string;
+  dataText?: string;
+  title?: string;
+  children?: React.ReactNode;
+}): JSX.Element => {
+  const spanRef = useRef<HTMLSpanElement>(null);
+  const aRef = useRef<HTMLAnchorElement>(null);
+
+  useEffect(() => {
+    // Apply github-buttons's render on a new span
+    if (spanRef.current && aRef.current) {
```

Comment:
I forgot to enforce it in the demo's linter,  but we usually prefer being explicit about what we check for - for example here checking explicitely for `spanRef.current !== null` instead - to avoid mistakenly having a false negative due to a falsy value (though I don't think that there's one problematic case here with the current React API)

Reply:
Done!